test/credentials_obfuscation_pbe_SUITE.erl

```diff
@@ -8,17 +8,21 @@
 -module(credentials_obfuscation_pbe_SUITE).
 -include_lib("common_test/include/ct.hrl").
 -compile(export_all).
- 
+
+%% This cipher is listed as supported on macOS, but doesn't actually work.
+%% OTP bug: https://bugs.erlang.org/browse/ERL-1478
+-define(SKIPPED_CIPHERS, [aes_ige256]).
```

Comment:
Good catch, thank you. I have added the same exception in `rabbit_common` because this cipher suite doesn't seem to be supported by some reasonably recent `libcrypto` versions on Linux, not just macOS.